src/transitions/cond/MCCondSignal.cpp

```diff
@@ -76,6 +76,10 @@ MCCondSignal::dependentWith(const MCTransition *other) const
 void
 MCCondSignal::print() const
 {
-  printf("thread %lu: pthread_cond_signal(cond:%u)\n", this->thread->tid,
-         countVisibleObjectsOfType(this->conditionVariable->getObjectId()));
+  const char * isLostWakeup = " [No thread waiting on cond; lost wakeup?]";
+  if (hasWaiters) { isLostWakeup = ""; }
```

Comment:
I think `hadWaiters` is ever so slightly clearer since we're talking about whether there were threads waiting on the condition variable _at the time the transition was created_, but we may be asking for a print out at some point later in the future.